### PR TITLE
Add release notes for cs-engine 1.12.6-cs8

### PR DESCRIPTION
**Pull request by cpuguy83**
Signed-off-by: Brian Goff <cpuguy83@gmail.com>

### CODE REVIEW

**Comment by mstanleyjones** [comment]
@cpuguy83 PTAL at my edits and LMK if you have questions.

File: cs-engine/1.12/install.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ to update its RHEL kernel.
     This adds the repository of the latest version of CS Docker Engine. You can
     customize the URL to install an older version.
 
-    *Note*: For users on RHEL 7.2 who have issues with installing the selinux
-    please use the following command instead of the one above:
+    > **Note**: For users on RHEL 7.2 who have issues with installing selinux,
```
Comment (cpuguy83) [inline cs-engine/1.12/install.md]: "the selinux policy" is what I intended here.

Comment (mstanleyjones) [inline cs-engine/1.12/install.md]: I changed it to that.

**Comment by cpuguy83** [comment]
Changes look good, thanks!

**Pull request merged**